src/cirrus/release.py

```diff
@@ -193,7 +208,7 @@ def new_release(opts):
 
     # update cirrus conf
     config.update_package_version(new_version)
-    changes = ['cirrus.conf']
+    changes = ['cirrus.conf', 'requirements.txt']
```

Comment:
I think you want to add this in the if clause only if the file is modified by the bump, right?